straxen/plugins/event_processing.py

```diff
@@ -727,46 +728,56 @@ def compute(self, events):
 
 
 @export
-@strax.takes_config(
-    strax.Option(
-        'g1', infer_type=False,
-        help="S1 gain in PE / photons produced",
-        default_by_run=[(0, 0.1442),
-                        (first_sr1_run, 0.1426)]),
-    strax.Option(
-        'g2', infer_type=False,
-        help="S2 gain in PE / electrons produced",
-        default_by_run=[(0, 11.52/(1 - 0.63)),
-                        (first_sr1_run, 11.55/(1 - 0.63))]),
-    strax.Option(
-        'lxe_w', infer_type=False,
-        help="LXe work function in quanta/keV",
-        default=13.7e-3),
-)
 class EnergyEstimates(strax.Plugin):
     """
-    Plugin which converts cS1 and cS2 into energies (from PE to KeVee).
+    Plugin which converts cS1 and cS2 into energies (from PE to KeVee) and quanta.
     """
-    __version__ = '0.1.0'
+    __version__ = '0.2.0'
     depends_on = ['corrected_areas']
-    dtype = [
-        ('e_light', np.float32, 'Energy in light signal [keVee]'),
-        ('e_charge', np.float32, 'Energy in charge signal [keVee]'),
-        ('e_ces', np.float32, 'Energy estimate [keVee]')
-    ] + strax.time_fields
+    dtype = [('e_light', np.float32, 'Energy in light signal [keVee]'),
+             ('e_charge', np.float32, 'Energy in charge signal [keVee]'),
+             ('e_ces', np.float32, 'Energy estimate [keVee]'),
+             ('n_electrons', np.float32, 'Estimated number of electrons produced'),
+             ('n_photons', np.float32, 'Estimated number of photons produced'),
```

Comment:
perhaps add that (for the n_photons) you mean prompt scintillation photons produced in the S1 (just to avoid confusion with the S2 also producing photons)